Oracle_CQRS/src/main/java/org/example/oracle_cqrs/command/controlles/AccountCommandRest.java

```diff
@@ -0,0 +1,68 @@
+package org.example.oracle_cqrs.command.controlles;
+
+
+import org.example.oracle_cqrs.command.commands.CreateAccountCommand;
+import org.example.oracle_cqrs.command.commands.CreditAccountCommand;
+import org.example.oracle_cqrs.command.commands.DebitAccountCommand;
+import org.example.oracle_cqrs.command.commands.UpdateAccountStatusCommand;
+import org.example.oracle_cqrs.command.producers.CommandsProducer;
+import org.example.oracle_cqrs.command.repository.EventStoreRepository;
+import org.example.oracle_cqrs.common.Dtos.CreateAccountDTO;
+import org.example.oracle_cqrs.common.Dtos.CreditAccountDTO;
+import org.example.oracle_cqrs.common.Dtos.DebitAccountDTO;
+import org.example.oracle_cqrs.common.Dtos.UpdateAccountStatusDTO;
+import org.example.oracle_cqrs.common.events.BaseEvent;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/api/commands")
+public class AccountCommandRest {
+
+    private CommandsProducer commandsProducer ;
+    private EventStoreRepository eventStoreRepository;
+
+    AccountCommandRest(CommandsProducer commandsProducer, EventStoreRepository eventStoreRepository) {
+        this.commandsProducer = commandsProducer;
+        this.eventStoreRepository = eventStoreRepository;
+    }
+
+    @PostMapping("/create")
+    public void createAccount(@RequestBody CreateAccountDTO request){
```

Comment:
these are where command validation can occur, before we try to place on the queue

Reply:
suggest also to return 201 CREATED instead of void